lib/schema/physicalActivity.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2020, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+'use strict';
+
+var schema = require('./schema.js');
+
+var idFields = ['type', 'deviceId', 'time'];
+schema.registerIdFields('physicalActivity', idFields);
+
+module.exports = schema.makeHandler('physicalActivity', {
+  schema: {
+    duration: schema.and(schema.isObject, schema.ensureSchemaFn('duration', {
+      value: schema.isNumber,
```

Comment:
This doesn't quite [match the behaviour in platform](https://github.com/tidepool-org/platform/blob/25bfa5aaf9702b1d6a0eebe08798285c33868183/data/types/activity/physical/duration.go#L60-L72), which has more validation around the values depending on the units. If we wanted to replicate this behaviour, we may have to do a higher-level `schema.or`.